content/en/docs/reference/access-authn-authz/node.md

```diff
@@ -34,7 +34,7 @@ Write operations:
 
 Auth-related operations:
 
-* read/write access to the certificationsigningrequests API for TLS bootstrapping
```

Comment:
In the API itself, the path contains `/certificatesigningrequests/`. If we change this line only, the next line is incongruous.

It'd be OK to instead link to the API references for these three APIs.